### PR TITLE
Added support for region overwrite

### PR DESCRIPTION
**Pull request by zdmytriv**
```
aws-profile -r us-east-1 aws configure list

# returns
   profile                      dev           manual    --profile
access_key     ****************PMHB              env
secret_key     ****************pK/7              env
    region                us-east-1              env    AWS_DEFAULT_REGION
```

### CODE REVIEW

**Comment by jrstarke** [comment]
Why is this necessary when the AWS command has the`--region` flag?

**Comment by zdmytriv** [comment]
```
AWS_PROFILE=dev aws-profile terraform plan
```
can't pass region into command like this

**Comment by jrstarke** [comment]
If it's always the same region, you can set it on the profile. There is also `AWS_DEFAULT_REGION`

```
AWS_PROFILE=dev AWS_DEFAULT_PROFILE=us-east-1 aws-profile terraform plan
```

But I agree that's pretty clunky.

Looks like the tests are failing on your branch, but are still succeeding in master (https://travis-ci.org/github/jrstarke/aws-profile/jobs/661920493)

**Comment by zdmytriv** [comment]
I'll fix tests

**Comment by zdmytriv** [comment]
fixed tests, please review

File: setup.py [hunk 1]
```diff
@@ -8,8 +8,8 @@
 here = path.abspath(path.dirname(__file__))
 
 requires = [
-    'botocore>=1.3.15',
-    'awscli>=1.14.10'
+    'botocore>=1.15.20',
+    'awscli>=1.18.20'
```
Comment (jrstarke) [inline setup.py]: Unless I'm mistaken, nothing you've added here requires a newer version of either the awscli or boto. If you have a more recent version, this should still work with them fine.

Comment (zdmytriv) [reply setup.py]: you are right. reverted this change.

**Comment by zdmytriv** [comment]
@jrstarke ping

**Review by jrstarke — APPROVED** [review]

**Comment by jrstarke** [comment]
@zdmytriv This has been released as [v0.6.0](https://github.com/jrstarke/aws-profile/releases/tag/v0.6.0). This has been pushed up to PyPI so you should be able to install it from there.

Sorry for the slow turn around. things have been a bit crazy the last few weeks.

**Comment by zdmytriv** [comment]
Awesome, thanks @jrstarke 👍

**Pull request merged**